docs/documentation/server_admin/topics/clients/oidc/con-basic-settings.adoc

```diff
@@ -48,14 +48,16 @@ the legacy {project_name} Java OIDC adapters or by the link:https://docs.wildfly
 == Capability Config
 [[_access-type]]
 
-*Client authentication*:: The type of OIDC client.
+*Client authentication*:: Specifies the type of OIDC client.
 * _ON_
 +
 For server-side clients that perform browser logins and require client secrets when making an Access Token Request. This setting should be used for server-side applications.
+Clients with the client authentication enabled are referred as confidential clients. For more details, see <<_client-credentials, Client credentials>>.
 
 * _OFF_
 +
 For client-side clients that perform browser logins. As it is not possible to ensure that secrets can be kept safe with client-side clients, it is important to restrict access by configuring correct redirect URIs.
+Clients with the client authentication enabled are referred as public clients.
```

Comment:
@mposolda  Apologies for the late review and interruption. Did we actually mean to say "disabled" here instead of "enabled"?. "Clients with client authentication disabled are referred to as public clients."